### DISCUSSION TITLE
Calendar value - is it UTC?

### DISCUSSION THREAD

**Discussion by bhaidar** [primefaces/community#407] [PrimeVue]
Hi,

I am using the Calendar control. I select one value and I see another value. The value is in the ISO-8601 format. Does the calendar return a value in UTC or local datetime?

<img width="391" alt="image" src="https://github.com/primefaces/community/assets/1163421/f59ae158-9f89-4a47-aa29-e5a4aabb9ef2">

However, when I print the InertiaJS form value, I get this: (This is the data to be sent to the server)
`"end_date": "2023-11-19T15:39:00.000Z"`

My question is, is the date value sent in UTC to the server?

Thank you

**Comment by tugcekucukoglu** [comment]
Hi,

Calendar does not change the data that Date() constructor returned. Shows formatted value to the user. You need to format the data you send to the server yourself.

**Reply by bhaidar** [reply]
Thanks for your feedback. Can you explain why then I get the following results?

I select 12:08 as time, but then in the data printed it's 10:08

<img width="447" alt="image" src="https://github.com/primefaces/community/assets/1163421/9c166338-0ad5-4dc7-a991-a937838390b6">

**Reply by bhaidar** [reply]
Nevermind, Vue displays objects as JSON and the Date.toJSON makes use of the ISO8601 format. Solved :-)
